src/Components/ListOfArticles.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Grid, Container } from 'semantic-ui-react'
+
+class ListOfArticles extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      articles: []
```

Comment:
You can avoid the constructor if you don't have any specific functions in it. If you only need state you can simply write :
`state={articles: [ ]}`

Reply:
done